sig-windows/annual-report-2020

```diff
@@ -0,0 +1,69 @@
+# Operational 
```

Comment:
please rename to a .md suffix so this renders as markdown

Reply:
once you do that, view in the markdown rendering and add newlines (and maybe bold the questions) to make the answers legible and separate from the questions.